hr_holidays_public/models/hr_holidays_public.py

```diff
@@ -57,8 +62,7 @@ def _compute_display_name(self):
             if line.country_id:
                 line.display_name = '%s (%s)' % (
                     line.year,
-                    line.country_id.name
-                )
+                    line.country_id.name)
```

Comment:
Don't change ending characters deindented in other line. This is for readability, as you are doing for example in https://github.com/OCA/hr/pull/649/commits/a23796e805c8b65e228c90190bf43532263c1a69#diff-3080bf617f93d891bde828f3edd165d3R95

Reply:
Yeap, don't even know why I did that